src/event.js

```diff
@@ -226,6 +226,63 @@ geo_event.unzoomselect = 'geo_unzoomselect';
 
 //////////////////////////////////////////////////////////////////////////////
 /**
+ * Triggered when an action is initiated with mouse down
+ *
+ * @property {object} state The action state
+ * @property {geo.mouseState} mouse The mouse state
+ * @property {object} event The triggering event
+ */
+//////////////////////////////////////////////////////////////////////////////
+geo_event.actiondown = 'geo_actiondown';
```

Comment:
@manthey nitpick: we use camelcase style for other event for example 

geo_event.worldChanged = 'geo_worldChanged';

Should we change geo_event.actiondown to geo_event.actionDown? (and similarly for others too?)